### ISSUE TITLE
T-ray messes with the tool sprite layering on tool belt.

### ISSUE THREAD

**Issue by Sayrbee** [ParadiseSS13/Paradise#19336]
**Issue Description**:

having a T-ray on your belt makes some tool icons (Jaws of Life and Power Drill, probably others too) placed behind the belt when they should be in front.
(also I don't think the t-ray even has a belt sprite)
**What did you expect to happen**: 

the sprite to stay the same.

**What happened instead**:

I put a t-ray scanner on my belt and the sprites for the jaws of life and power drill and moved behind the toolbelt sprite when they used to be in front.

**Why is this bad/What are the consequences**:

It makes me hesitant to get advanced tools or not carry a t-ray scanner when I now I space for it on my belt.

**Steps to reproduce the problem**:

Put a set of advanced tools on your belt: jaws of life, power drill, experimental welder, multitool, wire. Then put a t-ray scanner on your belt. the sprites for the tools on the front side of the belt sprite should now be layered behind the belt.

**When did the problem start happening**:

I think it probably started when we added a bunch of new sprites for items on belts.

**Extra information**:
